src/h5web/toolbar/controls/DomainSlider/DomainSlider.tsx

```diff
@@ -50,20 +50,20 @@ function DomainSlider(props: Props) {
     toggleEditingMax(force);
   }
 
-  const rootRef = useRef(null);
-  const tooltipRef = useRef<DomainTooltipHandle>(null);
-
-  useClickOutside(rootRef, () => {
+  function cancelEditing() {
```

Comment:
Introducing a `cancelEditing` function called from both `useClickOutside` and `useKey` below. The goal is to avoid canceling the editing unnecessarily when pressing `Escape`. I did this to fix a test at first, but I'm not sure it's still needed. Regardless, I think the refactoring makes sense.